docs/scos/dev/feature-integration-guides/202304.0/glue-api/glue-api-product-rating-and-reviews-feature-integration.md

```diff
@@ -0,0 +1,666 @@
+            },
+            "links": {
+                "self": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews/23"
+            }
+        },
+        {
+            "type": "product-reviews",
+            "id": "25",
+            "attributes": {
+                "rating": 3,
+                "nickname": "Spencor",
+                "summary": "Curabitur ultricies, sapien quis placerat lacinia",
+                "description": "Etiam venenatis sit amet lorem eget tristique. Donec rutrum massa nec commodo cursus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Suspendisse scelerisque scelerisque augue eget condimentum. Quisque quis arcu consequat, lacinia nulla tempor, venenatis ante. In ullamcorper, orci sit amet tempus tincidunt, massa augue molestie enim, in finibus metus odio at purus. Mauris ut semper sem, a ornare sapien. Fusce eget facilisis felis. Integer imperdiet massa a tortor varius, tincidunt laoreet ipsum viverra."
+            },
+            "links": {
+                "self": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews/25"
+            }
+        },
+        {
+            "type": "product-reviews",
+            "id": "26",
+            "attributes": {
+                "rating": 5,
+                "nickname": "Spencor",
+                "summary": "Cras porttitor",
+                "description": "Cras porttitor, odio vel ultricies commodo, erat turpis pulvinar turpis, id faucibus dolor odio a tellus. Mauris et nibh tempus, convallis ipsum luctus, mollis risus. Donec molestie orci ante, id tristique diam interdum eget. Praesent erat neque, sollicitudin sit amet pellentesque eget, gravida in lectus. Donec ultrices, nisl in laoreet ultrices, nunc enim lacinia felis, ac convallis tortor ligula non eros. Morbi semper ipsum non elit mollis, non commodo arcu porta. Mauris tincidunt purus rutrum erat ornare, varius egestas eros eleifend."
+            },
+            "links": {
+                "self": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews/26"
+            }
+        }
+    ],
+    "links": {
+        "self": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5",
+        "last": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5",
+        "first": "http://glue.de.suite-nonsplit.local/abstract-products/139/product-reviews?page[offset]=0&page[limit]=5"
+    }
+}
+```
+</details>
+
+{% endinfo_block %}
+
+#### Enable query expanders
+Activate the following plugins:
+
+| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
+| --- | --- | --- | --- |
+| FilterByIdProductReviewQueryExpanderPlugin |  | None | Spryker\Client\ProductReviewSearch\Plugin\Search |
```

Comment:
it's not related to GLUE API feature/IG, it's a part of `Product rating & reviews feature` integration .

Reply:
The plugin was removed from this IG, _product-rating-and-reviews-feature-integration_ IG is pretty outdated and should be recreated considering a new template. FilterByIdProductReviewQueryExpanderPlugin should be mentioned there.